data/recipes.yaml

```diff
@@ -9,6 +9,11 @@
   out:
   - [4, board]
 
+- in:
+  - [100, log]
```

Comment:
I hope this is a joke otherwise I might need to go cut down a forest or two :fearful: :evergreen_tree: :evergreen_tree: :evergreen_tree: